techmatters/webchat
# Style modules refactor and adjust responsiveness for Quick Exit elements

Primary Reviewer: @murilovmachado 

## Description
- Change Quick Exit wrapper to be responsive for translations / longer strings
- Style modules refactor

### Checklist
- [n/a] New tests added



### Related Issues
Fixes # [CHI-1545](https://tech-matters.atlassian.net/browse/CHI-1545)
Check this conversation: https://tech-matters.slack.com/archives/CV4QM23UJ/p1671049407715749

### Verification steps
- Change `QuickExitButtonLabel` in `src/translations/en.ts` to a longer string
- Check app for `QuickExitButtonLabel`
